proposals/lip-0022.md

```diff
@@ -149,8 +149,8 @@ A round is composed of 103 blocks now.
 
 Before being ordered according to [LIP 0003](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0003.md) to generate the forging list, the set of forging delegates for `round` + 1 is constructed as follows:
 
-* The addresses of the top 101 delegates by delegate weight at the end of `round` - 2 are added to the set (the 2 rounds delay is specified in [Change of Delegates](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#change-of-delegates) subsection of LIP 0014). Ties are broken in favor of the smaller address.
-* The addresses of `delegate1` and `delegate2` are included to the set of forging delegates. These two delegates are the output of the selection algorithm specified in the next section.
+1. From the set of all non-banned delegates at the end of `round` - 2, the addresses of the top 101 delegates by delegate weight are added to set of forging delegates (the 2 rounds delay is specified in [Change of Delegates](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#change-of-delegates) subsection of LIP 0014). Ties are broken in favor of the smaller address. In case there are 103 or less delegates to select from, all non-banned delegates at the end of `round` - 2 are added to this set and the following step of adding 2 extra delegates is omitted.
```

Comment:
```suggestion
1. The addresses of the top 101 delegates by delegate weight are added to the set of forging delegates from the set of all non-banned delegates at the end of `round` - 2 (the 2 rounds delay is specified in [Change of Delegates](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#change-of-delegates) subsection of LIP 0014). Ties are broken in favor of the smaller address. In case there are 103 or less delegates to select from, all non-banned delegates at the end of `round` - 2 are added to this set and the following step of adding 2 extra delegates is omitted.
```